src/components/Comments/ReplyCommentItem.js

```diff
@@ -46,6 +55,15 @@ const ReplyCommentItem = ({
     comment.data.count_upvote - comment.data.count_downvote
   );
   const [statusVote, setStatusVote] = React.useState('');
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new loading state has been introduced for handling direct messages (DMs). This is a good practice as it helps in providing feedback to the user about the ongoing process. However, ensure that this loading state is properly reset in all scenarios including error cases.



<!-- This is an auto-generated comment by OSS CodeRabbit -->